supervisor/homeassistant.py

```diff
@@ -359,6 +359,13 @@ async def _update(to_version: str) -> None:
         # Update going wrong, revert it
         if self.error_state and rollback:
             _LOGGER.critical("HomeAssistant update fails -> rollback!")
+            # Make a copy of the current log file if it exsist
+            logfile = Path(self.sys_config.path_homeassistant, "home-assistant.log")
```

Comment:
If `path_homeassistant` is a Path object, this works too:
```suggestion
            logfile = self.sys_config.path_homeassistant / "home-assistant.log"
```